python_modules/libraries/dagstermill/setup.py

```diff
@@ -47,11 +47,11 @@ def get_version() -> str:
         "packaging>=20.5",
         "papermill>=1.0.0",
         "scrapbook>=0.5.0",
+        "nbconvert",
```

Comment:
rendering the notebook in dagit seems like a pretty nice feature to include by default, @yuhan what do you think of having nbconvert as a base dependency?

Reply:
there used to be some conflict dep. but if all the builds are running fine, i think it's fine